dev/integration_tests/android_semantics_testing/lib/main.dart

```diff
@@ -8,22 +8,19 @@ import 'dart:convert';
 import 'package:flutter/material.dart';
 import 'package:flutter/scheduler.dart';
 import 'package:flutter/services.dart';
-import 'package:flutter_driver/driver_extension.dart';
 
 import 'src/tests/controls_page.dart';
 import 'src/tests/headings_page.dart';
 import 'src/tests/popup_page.dart';
 import 'src/tests/text_field_page.dart';
 
 void main() {
-  timeDilation = 0.05; // remove animations.
-  enableFlutterDriverExtension(handler: dataHandler);
   runApp(const TestApp());
 }
 
 const MethodChannel kSemanticsChannel = MethodChannel('semantics');
 
-Future<String> dataHandler(String? message) async {
+Future<String> invokeSemanticsMethod(String? message) async {
```

Comment:
This method could probably just live in the test file now and be broken up in separate methods for getSemanticsNode and setClipboard? Also, the arguments can be passed directly, no need to turn them into a string `message`?

Reply:
Also, there shouldn't be a need to serialize the return value?

Reply:
refactored